WtWSMS/localization/english/bookmark/bm_476_l_english.yml

```diff
@@ -2,84 +2,84 @@
 
- bookmark_476_romulus_augustulus:0 "Romulus"
- bookmark_476_romulus_augustulus_desc:0 "Proclaimed as emperor by his father, Romulus Augustus was little more than a child. He had usurped the Western Roman Empire, which had been crippled by centuries of strife and decline. However, with the barbarian Odovacar threatening his already uneasy and short reign, what fate lies for the young emperor?"
+ bookmark_476_romulus_augustulus: "Romulus"
+ bookmark_476_romulus_augustulus_desc: "Proclaimed as emperor by his father, Romulus Augustus was little more than a child thrust into Emperorship. He, via the will of his father, usurped the Western Roman Empire, which had been crippled by centuries of strife and decline. With rise of the barbarian general Odovacar threatening his already uneasy and short reign, what fate lies for the young emperor?\n\n#bold Can the child Emperor be the second coming of Romulus, or is his name a portent of the end of the #italic Pars Occidentalis#!?#!"
 
- bookmark_476_romulus_augustulus_alt_orestes:0 "Orestes"
- bookmark_476_romulus_augustulus_alt_orestes_desc:0 "alt desc"
+ bookmark_476_romulus_augustulus_alt_orestes: "Orestes"
+ bookmark_476_romulus_augustulus_alt_orestes_desc: "alt desc"
  
- BOOKMARK_RELATION_FATHER:0 "(Father)"
- BOOKMARK_RELATION_FATHER_REGENT:0 "(Father and Regent)"
+ BOOKMARK_RELATION_FATHER: "(Father)"
+ BOOKMARK_RELATION_FATHER_REGENT: "(Father and Regent)"
  
- bookmark_476_theodoric:0 "Theodoric"
- bookmark_476_theodoric_desc:0 "Theodoric, raised a hostage of the Eastern Roman Empire, was raised a Roman and received a complete education. In 473, he would become the leader of the Ostrogoths of Pannonia, and less than ten years later, unite the Ostrogoths under his rule. However, his greatest achievement would be to overthrow Odovacar and establish an Ostrogothic Kingdom across the entire Italian Peninsula. Vying for a return to greatness, like that under the ancient Roman Empire, will his dreams come to fruition?"
+ bookmark_476_theodoric: "Theodoric"
+ bookmark_476_theodoric_desc: "Theodoric, raised a hostage of the Eastern Roman Empire, received a complete Romanizing Education. In 473, he would become the leader of the Ostrogoths of Pannonia, and less than ten years later, unite all of the Eastern Goths under his rule. However, his greatest achievement would be to overthrow Odovacar and establish an Ostrogothic Kingdom across the entire Italian Peninsula.\n\n#bold Seeking to rise to greatness, like the ancient Romans, will the #italic People-Ruler's#! dreams come to fruition?#!"
```

Comment:
Just "Roman education", no? And why "People-Ruler", I don't get the reference.

Reply:
Theodoric, like a lot of Germanic names is a 2-part name; in this case Thiuda (People) and Reiks (Ruler), so People-Ruler is a literal translation of this name. Maybe I'm leaning into nominal etymology a bit too much.